### DISCUSSION TITLE
Dead, dead, dead - someone please take over?

### DISCUSSION THREAD

**Discussion by HyperSane** [deathau/markdownload#353] [General]
This was once a great extension, sadly has been discontinued for over 2 YEARS with zero communication.
Issues that have existed for over a year are still present.

Can someone please make a similar app or take over or something?

**Comment by SamejSpenser** [comment]
> (…) has been discontinued for over 2 YEARS with zero communication…

What do you mean, _“has been discontinued for over 2 YEARS with zero communication”_? The latest version, v3.4.0, is from less than two months ago, on August 23.

Have you checked this out? 🤔

“[Markdownload's future #342](https://github.com/deathau/markdownload/discussions/342)”
